docs/en/latest/plugins/limit-conn.md

```diff
@@ -141,3 +141,76 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
     }
 }'
 ```
+
+## Limit the rate of WebSocket connections
```

Comment:
```suggestion
## Limit connections of WebSocket
```

Reply:
How about `Limit the number of concurrent WebSocket connections` ?